library_generation/test/test_utilities.sh

```diff
@@ -281,23 +281,20 @@ get_version_from_WORKSPACE() {
   echo "${version}"
 }
 
-# Convenience function to clone only the necessary folders from a git repository
-sparse_clone() {
-  repo_url=$1
-  paths=$2
-  commitish=$3
-  clone_dir=$(basename "${repo_url%.*}")
-  rm -rf "${clone_dir}"
-  git clone -n --depth=1 --no-single-branch --filter=tree:0 "${repo_url}"
-  pushd "${clone_dir}"
-  if [ -n "${commitish}" ]; then
-    git checkout "${commitish}"
-  fi
-  git sparse-checkout set --no-cone ${paths}
-  git checkout
-  popd
+download_googleapis_files_and_folders() {
+  local output_folder=$1
+  # checkout the master branch of googleapis/google (proto files) and WORKSPACE
+  echo "Checking out googlapis repository..."
+  # sparse_clone will remove folder contents first, so we have to checkout googleapis
+  # only once.
+  sparse_clone https://github.com/googleapis/googleapis.git "google grafeas WORKSPACE"
```

Comment:
Do we need `WORKSPACE` anymore?

Reply:
We do https://github.com/googleapis/sdk-platform-java/blob/ed16261ac8c90fe588026e30d62d948e7ca1af13/library_generation/test/generate_library_integration_test.sh#L66

The IT relies on the WORKSPACE to infer a couple versions

Reply:
Can we fix the generator and protobuf version for the integration test?

Reply:
On a second thought, fixing generator version should go to a separate PR (if any).